packages/bpk-component-swap-button/src/BpkSwapButton.module.scss

```diff
@@ -19,8 +19,6 @@
 @import '../../bpk-mixins/index.scss';
 
 .bpk-swap-button {
-  width: bpk-spacing-xxl();
-  height: bpk-spacing-xxl();
```

Comment:
The widths here didn't really do anything and causing issues with the width of the button. Moving these values onto the button element instead